spec/acceptance/httpclient/httpclient_spec.rb

```diff
@@ -113,13 +113,8 @@ def filter_response(request, response)
         nil # to let the request be made for real
       end
 
-      # To make two requests that have the same request signature, the headers must match.
-      # Since the webmock server has a Set-Cookie header, the 2nd request will automatically
-      # include a Cookie header (due to how httpclient works), so we have to set the header
-      # manually on the first request but not on the 2nd request.
-      http_request(:get, webmock_server_url, :client => client,
-                         :headers => { "Cookie" => "bar=; foo=" })
-      http_request(:get, webmock_server_url, :client => client)
+      http_request(:get, webmock_server_url, :client => client, :headers => { "Cookie" => "bar=; foo=" })
```

Comment:
looks like I spoke too soon, I have now realized this fails if I run without bundle install, but works with. And the way I have my vim setup to run focused tests uses bundle exec, but I was running the whole suite with just rspec.

So I'm now trying to track down the dependency difference that is causing this to fail.